src/core/border_router/routing_manager.hpp

```diff
@@ -556,6 +559,20 @@ class RoutingManager : public InstanceLocator
      *
      */
     Dhcp6PdState GetDhcp6PdState(void) const { return mPdPrefixManager.GetState(); }
+
+    /**
+     * Sets the callback to notify when the state of request/release a prefix via DHCPv6 Prefix Delegation (PD) is
+     * changed.
+     *
+     * @param[in] aCallback  A pointer to a function that is called when request/release PD state changes.
```

Comment:
@param[in]  aCallback   A pointer to a function that is called whenever the state of a prefix request or release changes.
```suggestion
     * @param[in] aCallback  A pointer to a function that is called when request/release PD state changes.
```